server/src/modules/refresh-token/refresh-tokens.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { RefreshTokenEntity } from './refresh-token.entity';
+import { refreshTokensServiceErrorMessages } from './refresh-tokens.constants';
+import { BaseService } from 'src/common/services';
+import * as ms from 'ms';
+import { AppConfigService } from 'src/config/app-config.service';
+import { UsersService } from '../users/users.service';
+
+@Injectable()
+export class RefreshTokensService extends BaseService<RefreshTokenEntity> {
+  constructor(
+    @InjectRepository(RefreshTokenEntity)
+    private readonly refreshTokenEntityRepository: Repository<RefreshTokenEntity>,
+    private appConfigService: AppConfigService,
```

Comment:
It's good practice to mark your injected services as readonly to prevent implicit bugs
```suggestion
    private readonly appConfigService: AppConfigService,
```